cell.go

```diff
@@ -494,6 +494,57 @@ func (f *File) SetCellHyperLink(sheet, axis, link, linkType string) error {
 	return nil
 }
 
+// GetCellRichText provides a function to get rich text of cell by given
+// worksheet.
+func (f *File) GetCellRichText(sheet, cell string) (runs []RichTextRun, err error) {
+	ws, err := f.workSheetReader(sheet)
+	if err != nil {
+		return
+	}
+	cellData, _, _, err := f.prepareCell(ws, sheet, cell)
+	if err != nil {
+		return
+	}
+
+	siIndex, err := strconv.Atoi(cellData.V)
+	if nil != err {
+		return
+	}
+
+	sst := f.sharedStringsReader()
+	a := sst.SI[siIndex]
+	for _, v := range a.R {
+		run := RichTextRun{
+			Text: v.T.Val,
+		}
+		if nil != v.RPr {
+			font := Font{}
+			font.Bold = v.RPr.B == " "
```

Comment:
Different with set the rich text, by OOXML spec, we need also handle italic and bold style created by Excel and other application, which maybe will not have an empty string, and just have a self closed XML element such as `</b>`, `</i>` or `</strike>`, etc, I purpose to change the data type of following fields in the `xlsxRPr` structure to a pointer of string like this:

```go
B         *string         `xml:"b"`
I         *string         `xml:"i"`
Strike    *string         `xml:"strike"`
Outline   *string         `xml:"outline"`
Shadow    *string         `xml:"shadow"`
Condense  *string         `xml:"condense"`
Extend    *string         `xml:"extend"` 
```

you know this change will affect the set rich text.